app/mailers/support_mailer.rb

```diff
@@ -11,10 +11,11 @@ def confirm_sign_in(support_user, device:)
 
   def fallback_sign_in_email(support_user, token)
     @token = token
+    subject_key = FeatureFlag.active?(:continuous_applications) ? :continuous_application_subject : :subject
```

Comment:
Same above

Reply:
Actually, we might not have application form here? Need to understand this method. But if you understand the above we might need to change this.